doc/ipad.md

```diff
@@ -28,6 +30,31 @@
   - Alternative: Just use touch scrolling
 - See [issues tagged with the iPad label](https://github.com/cdr/code-server/issues?q=is%3Aopen+is%3Aissue+label%3AiPad) for more.
 
+## How to install PWA
+
+To install the code-server PWA, follow these steps:
+
+1. Open code-server in Safari
+2. Click the Share icon
+3. Click "Add to Home Screen"
+
+Now when you open code-server from the home screen, you will be using the PWA.
+The advantages of this are more screen real estate and access to top-level keyboard shortcuts because it's running like an app.
+An example shortcut is the `cmd+w` to close an active file in the workbench. You can add this to your `keybindings.json` by doing the following:
```

Comment:
Is this okay? If not, we should update the `yarn fmt` script to hard-wrap to the line length we want.

Reply:
As long as CI does not complain, I think it's fine in my book!